.github/workflows/linter.yml

```
@@ -0,0 +1,27 @@
+name: Lint Code Base
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches: main
+permissions: read-all
+jobs:
+  build:
+    name: Lint Code Base
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          # Full git history is needed to get a proper
+          # list of changed files within `super-linter`
+          fetch-depth: 0
+      - name: Lint Code Base
+        uses: super-linter/super-linter@v6.3.0
+        env:
+          VALIDATE_ALL_CODEBASE: true
```

```suggestion
          VALIDATE_ALL_CODEBASE: true
          FILTER_REGEX_EXCLUDE: "/github/workspace/api_gateway/gradlew"
```
exclude auto-generated file

---

Let's not include the GH path in the filter:
```suggestion
          VALIDATE_ALL_CODEBASE: true
          FILTER_REGEX_EXCLUDE: "api_gateway/gradlew"
```

---

```suggestion
          VALIDATE_ALL_CODEBASE: true
          FILTER_REGEX_EXCLUDE: "*/api_gateway/gradlew"
```
Then it should be like this

---

That's not a valid regex, a dot is missing :)

Let's go with a simple:
```suggestion
          VALIDATE_ALL_CODEBASE: true
          FILTER_REGEX_EXCLUDE: ".*/gradlew"
```
We're going to need for all of the services anyway.